docs/howto.rst

```diff
@@ -75,3 +77,100 @@ For more complex crash sets, pass a search URL to generate the list::
     app@socorro:app$ socorro-cmd fetch_crashids --num=all --url="https://crash-stats.mozilla.org/search/?product=Sample&date=%3E%3D2019-05-07T22%3A00%3A00.000Z&date=%3C2019-05-07T23%3A00%3A00.000Z" > crashids
     app@socorro:app$ cat crashids | socorro-cmd reprocess
 
+
+Processing requests for PII access
+==================================
+
+People file bugs in Bugzilla asking to be granted access to PII.
```

Comment:
I think there should be a link to [Access to memory dumps](https://crash-stats.mozilla.com/documentation/memory_dump_access/). It would make sense in this opening paragraph, since it has the instructions for the requester, including the Bugzilla link to get it to the right place. Alternatively, link below where you mention the terms to copy and paste.